variables.tf

```diff
@@ -34,6 +34,8 @@ variable "records" {
     proxied:
       Whether the record gets Cloudflare's origin protection.
       Default value: false.
+    key:
```

Comment:
The documentation would need to be updated to showcase this change

Reply:
seems to me documentation generation was already outdated:
- https://github.com/cloudposse/terraform-cloudflare-zone/pull/54

Reply:
i'll rebase after https://github.com/cloudposse/terraform-cloudflare-zone/pull/54 is merged